dotnet/dotnet-guestbook/.readmes/intellij/README.md

```diff
@@ -21,17 +25,24 @@ The Guestbook sample demonstrates how to deploy a Kubernetes application with a
 ![Kubernetes Architecture Diagram](./img/dotnet-guestbook-k8s-diagram.png)
 
 ### Directory contents
-- `src/frontend` - Guestbook frontend service
-- `src/backend` - Guestbook backend service
-- `skaffold.yaml` - A schema file that defines skaffold configurations ([skaffold.yaml reference](https://skaffold.dev/docs/references/yaml/))
-- `kubernetes-manifests/` - Contains Kubernetes YAML files for the Guestbook services and deployments, including:
-
-  - `guestbook-frontend.deployment.yaml` - deploys a pod with the frontend container image
-  - `guestbook-frontend.service.yaml` - creates a load balancer and exposes the frontend service on an external IP in the cluster
-  - `guestbook-backend.deployment.yaml` - deploys a pod with the backend container image
-  - `guestbook-backend.service.yaml` - exposes the backend service on an internal IP in the cluster
-  - `guestbook-mongodb.deployment.yaml` - deploys a pod containing a MongoDB instance
-  - `guestbook-mongodb.service.yaml` - exposes the MongoDB service on an internal IP in the cluster
+- `src/frontend/` - Guestbook frontend service, containing the following config files:
```

Comment:
should we put the root skaffold.yaml on top of this listing and mention it serves as an entry point to all (both) modules of the app?